src/main/java/eiteam/esteemedinnovation/converter/TileEntityFluidSteamConverter.java

```diff
@@ -39,7 +39,7 @@
     public TileEntityFluidSteamConverter() {
         Fluid steam = FluidRegistry.getFluid("steam");
         if (steam == null && CrossMod.IC2) {
-            steam = FluidRegistry.getFluid("ic2steam");
+            //steam = FluidRegistry.getFluid("ic2steam");
```

Comment:
Why is this commented out?

Reply:
IC2's coremod was crashing so I removed it. Kinda commented this out as a precaution that it won't somehow crash here as well. (It shouldn't)